### PR TITLE
ContentVersion cleanup backoffice UI

### PR DESCRIPTION
**Pull request by p-m-j**

### CODE REVIEW

**Review by bergmania — APPROVED** [review]
LGTM 💪

**Comment by p-m-j** [comment]
:tada::coffee::arrow_double_up:

File: src/Umbraco.Core/Persistence/Repositories/Implement/DocumentVersionRepository.cs [hunk 1]
```diff
@@ -55,6 +62,47 @@ public IReadOnlyCollection<ContentVersionCleanupPolicySettings> GetCleanupPolici
...
+                .LeftJoin<UserDto>()
+                    .On<UserDto, ContentVersionDto>(left => left.Id, right => right.UserId)
+                .LeftJoin<ContentVersionCultureVariationDto>()
+                    .On<ContentVersionCultureVariationDto, ContentVersionDto>(left => left.VersionId, right => right.Id)
+                .Where<ContentVersionDto>(x => x.NodeId == contentId);
+
+            // TODO: If there's not a better way to write this then we need a better way to write this.
+            query = languageId.HasValue
+                ? query.Where<ContentVersionCultureVariationDto>(x => x.LanguageId == languageId.Value)
+                : query.Where("umbracoContentVersionCultureVariation.languageId is null");
```
Comment (bjarnef) [inline src/Umbraco.Core/Persistence/Repositories/Implement/DocumentVersionRepository.cs:95]: @bergmania maybe I am missing something, but shouldn't the second line use `Cms.Core.Constants.DatabaseSchema.Tables.ContentVersionCultureVariation`?
https://github.com/umbraco/Umbraco-CMS/blob/v9/contrib/src/Umbraco.Core/Persistence/Constants-DatabaseSchema.cs#L28

`query.Where<ContentVersionCultureVariationDto>(x => x.LanguageId  == null);`

or 

`query.Where($"{Cms.Core.Constants.DatabaseSchema.Tables.ContentVersionCultureVariation}.languageId is null");`

Comment (bjarnef) [reply src/Umbraco.Core/Persistence/Repositories/Implement/DocumentVersionRepository.cs:95]: Ahh, I missed the `umbraco` table prefix.
https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Core/Persistence/Constants-DatabaseSchema.cs#L28

Anyway I think it should use either the dto class or the constant.
https://github.com/umbraco/Umbraco-CMS/blob/v9/contrib/src/Umbraco.Core/Persistence/Constants-DatabaseSchema.cs#L28

Comment (p-m-j) [reply src/Umbraco.Core/Persistence/Repositories/Implement/DocumentVersionRepository.cs:95]: Can't do `query.Where<ContentVersionCultureVariationDto>(x => x.LanguageId == null);` as LanguageId is int.
Using constants sounds smnart.

We should probably just create a sql view and create a poco to represent that -- however sql ce

Comment (bjarnef) [reply src/Umbraco.Core/Persistence/Repositories/Implement/DocumentVersionRepository.cs:95]: Ahh okay, the constants seems a bit nicer then 😁

**Pull request merged**